raiden/tests/unit/transfer/test_channel.py

```diff
@@ -164,7 +166,8 @@ def make_unlock(unlock_end, partner_end):
 
 
 def test_is_balance_proof_usable_onchain_answer_is_false():
-    channel_state = factories.make_channel_set(number_of_channels=1).channels[0]
+    channel_state = factories.make_channel_set(number_of_channels=1, token_address=factories.UNIT_TRANSFER_DESCRIPTION.initiator).channels.get(factories.UNIT_TRANSFER_DESCRIPTION.initiator)
+    channel_state = channel_state.get(next(iter(channel_state)))
```

Comment:
This is also confusing. Channel state is an object that represents the information about a channel. It is a class. Why youre using a iterator? Also the `channel_state ` variable is defined twice here, its reasinged.